extension/src/stabilization_info.rs

```diff
@@ -11,6 +11,11 @@
 
 crate::functions_stabilized_at! {
     STABLE_FUNCTIONS
+    "1.13.0" => {
```

Comment:
Marked this as stable in 1.13.0 since it is not going in the upcoming release

Reply:
If it's not going into the upcoming release (we can actually discuss this since we pushed the release out to next week), shouldn't this be marked as stable in 1.14.0?